models/labels/addresses/social/identifier/ens/labels_ens.sql

```diff
@@ -19,10 +20,10 @@ FROM (
        'ENS' as category, --should be social but we can't change this due to how many queries it probably breaks.
        '0xRob' as contributor,
        'query' AS source,
-       date('2022-10-06') as created_at,
+       timestamp '2022-10-06' as created_at,
```

Comment:
are you changing this from `date` to `timestamp` so that downstream unions succeed?

Reply:
We may either use `date` or `timestamp` for a certain column for consistency.
But date can union with timestamp in this case, As date is a subtype of timestamp.

Reply:
I will change back to `date()`.